projects/example-app/src/app/core/effects/router.effects.ts

```diff
@@ -0,0 +1,34 @@
+import { Injectable } from '@angular/core';
+import { Router, NavigationEnd, ActivatedRoute } from '@angular/router';
+import { Title } from '@angular/platform-browser';
+
+import { tap, filter, map, mergeMap } from 'rxjs/operators';
+
+import { createEffect } from '@ngrx/effects';
+
+@Injectable()
+export class RouterEffects {
+  updateTitle$ = createEffect(
+    () =>
+      this.router.events.pipe(
```

Comment:
We have store-router package installed in the example app. Wouldn't that be better to listen to those changes?
@brandonroberts

Reply:
I thought about that when I was writing the code :) The idea was to show how one would create effects not based on the Actions stream, so I went for this instead.

Reply:
I think these are good for what we asked for